prober/dns.go

```diff
@@ -142,6 +142,10 @@ func ProbeDNS(ctx context.Context, target string, module config.Module, registry
 		Name: "probe_dns_additional_rrs",
 		Help: "Returns number of entries in the additional resource record list",
 	})
+	probeDNSTargetReachable := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_dns_target_reachable",
```

Comment:
Based on the issue, and what this represents, this isn't about the target being reachable. It's more about the query being answered or refused.

How about:
```suggestion
		Name: "probe_dns_query_succeeded",
```

Reply:
Thanks for the quick reply! 'probe_dns_query_succeeded' works for me.